src/ansys/fluent/core/logging_config.yaml

```diff
@@ -1,84 +1,50 @@
+# More examples and references:
+# https://docs.python.org/3/howto/logging-cookbook.html
+# https://docs.python.org/3/library/logging.html#logging-levels
+# https://gist.github.com/kingspp/9451566a5555fb022215ca2b7b802f19
+
 version: 1
-disable_existing_loggers: False
+disable_existing_loggers: no
+
 formatters:
-  simple:
-    format: "%(asctime)s %(name)-12s %(levelname)-8s %(message)s"
+  console_fmt:
+    format: "%(name)s %(levelname)s: %(message)s"
+  logfile_fmt:
+    format: "%(asctime)s %(name)-21s %(levelname)-8s %(message)s"
+
 handlers:
-  console:
+  console_stdout:
     class: logging.StreamHandler
-    level: INFO
-    formatter: simple
+    level: ERROR
+    formatter: console_fmt
     stream: ext://sys.stdout
-  fluent.file:
+  pyfluent_file:
     class: logging.handlers.RotatingFileHandler
-    level: DEBUG
-    formatter: simple
+    level: NOTSET #  no log level filtering on this handler, message log levels are managed by the loggers instead
+    formatter: logfile_fmt
     filename: pyfluent.log
```

Comment:
Shall we append the timestamp to the log filename? Then the log file for each session will be preserved (like Fluent's automatic transcript). TimedRotatingFileHandler can probably be used here.

Reply:
It did not work very well, and logging package does not seem to support YAML very well for this. Using the `TimedRotatingFileHandler` instead of the `RotatingFileHandler` also disables rollover on file size (it instead rotates based on time interval only), which means that we end up with some absolutely massive log files some times. Ideally we'd use a custom handler that is a mix of these two handlers. 

Sticking to `RotatingFileHandler` for now, we can try adding timestamp again later on. Feel free to take a look at what I'm doing and see if you can find a good way to implement `TimedRotatingFileHandler`, I have pushed my local changes to this branch.